salt/kubernetes-common/openstack-config.jinja

```diff
@@ -0,0 +1,16 @@
+[Global]
+auth-url={{ pillar['cloud']['config']['auth_url'] }}
+username={{ pillar['cloud']['config']['username'] }}
+password={{ pillar['cloud']['config']['password'] }}
+tenant-name={{ pillar['cloud']['config']['project'] }}
+domain-name={{ pillar['cloud']['config']['domain'] }}
+region={{ pillar['cloud']['config']['region'] }}
+[LoadBalancer]
+lb-version=v2
+subnet-id={{ pillar['cloud']['config']['subnet'] }}
```

Comment:
Oh, I just realized. This isn't necessarily enough, There are two configuration options for subnets:

    subnet-id = ....
    floating-network-id = ...

The first defines the internal network the LB will be on. And, in some clouds, this will be the only subnet needed. The second defines where to get a floating IP from, for clouds where the internal network isn't reachable. To work properly in all environments, we need to optionally be able to supply the second.

There is also security group related options, as depending on the backend LB driver used within Neutron, things behave radically differently. For example, the LBaaS v2 HAProxy driver will allow traffic by default.. However, the LBaaS v2 Octavia driver will not, all traffic will be blocked by defaut:

    manage-security-groups = true
    node-security-group = <some-id-here-that-is-the-secgroup-of-your-workers>